collect.py

```diff
@@ -170,6 +172,12 @@ def do_something(driver, elem_attributes=None):
             driver.execute_script("arguments[0].value = '#ff0000'", elem)
         else:
             raise Exception('Unsupported input type: %s' % input_type)
+    elif elem.tag_name == 'select':
+        elem.click()
```

Comment:
Do we need to click on the `select` element?

Reply:
I thought that we are trying to emulate user interaction with the webpage, hence I clicked the select element to display the select options. This is also the reason that I am not looking for 'option' elements directly. Please correct me if I am wrong.

Reply:
If it is necessary to click on the select element first before clicking on the option element, then it makes sense to do that.
If it isn't necessary, then there's no need to do it.

What happens if you make Selenium click on the option directly? Does it fail?

Reply:
I tried selecting the options directly and I observed  the following
1. It works fine on firefox with a glitch i.e. it is not selecting all options, rather it is revisitng some options which had previously been selected
2. It generates screenshots for the different options without actually selecting them hence leaving the select field blank (i.e. default selected option)

I have added all the relevant screenshots and the collect.py code here
https://gist.github.com/vikasmahato/58157977a4fc6d965ae6449fdbab37dc

Reply:
So the screenshots here are with the select click removed? It seems like we can't remove it then?

Reply:
No the screenshots were with using option directly. It appears that the elem.click() was unecessary and I have removed it and updated the pull request.

Reply:
Don't the screenshot show that in Chrome it doesn't work?